assets/app/scripts/controllers/overview.js

```diff
@@ -334,6 +355,11 @@ angular.module('openshiftConsole')
       }
     };
 
+    function isCustomHost(route) {
+      var defaultHost = route.serviceName + '-' + route.metadata.namespace;
```

Comment:
I guess we have TLS/non-TLS, and also path field, which could distinguish between routes with the same host.  I'm more concerned about generated routes for different services or different namespaces conflicting